cf-oscore/src/main/java/org/eclipse/californium/oscore/Decryptor.java

```diff
@@ -140,7 +140,7 @@ protected static byte[] decryptAndDecode(Encrypt0Message enc, Message message, O
 			plaintext = enc.decrypt(key);
 
 		} catch (CoseException e) {
-			LOGGER.error(ErrorDescriptions.DECRYPTION_FAILED + " " + e.getMessage());
+			LOGGER.error(ErrorDescriptions.DECRYPTION_FAILED + " {}", e.getMessage());
```

Comment:
The (theoretical) downside of this approach is:
It still creates a temporary string, the `ErrorDescriptions.DECRYPTION_FAILED + " {}"`.
And, if `ErrorDescriptions.DECRYPTION_FAILED` would contain some meta (`{}`), the it will fail either. Consider to use:

```
LOGGER.error("{} {}", ErrorDescriptions.DECRYPTION_FAILED, e.getMessage());
```

Reply:
Fixed in commit d75f91db7d1c68682787a92248a0e8ec91ad6ccf